### ISSUE TITLE
Django HP & ENE addresses inside Spiral Tower

### ISSUE THREAD

**Issue by xDrHellx** [xDrHellx/Bok-Interface#28]
Once inside Spiral Tower, Django's "Current HP" & "Current ENE" memory addresses becomes Sabata's.
For this specific dungeon, we should look at the memory addresses to use one more time & update them.

Outside of Spiral Tower, the memory addresses currently used are indeed Django's.

*During Sabata's section after the Remains immortal dungeon, these memory addresses updates Sabata's HP & ENE.
In this specific case, it seems to be "normal behavior", has there is only 1 active playable character at the time and you're forced to play as Sabata.*

**TLDR : Fix "Current HP" & "Current ENE" memory addresses within Spiral Tower.**
